makepkg/PKGBUILD

```diff
@@ -11,11 +11,12 @@ changelog=aurutils.changelog
 sha256sums=('SKIP')
 conflicts=('aurutils')
 provides=("aurutils=${pkgver%%.r*}")
-depends=('git' 'jq' 'pacutils' 'wget')
+depends=('git' 'jq' 'pacutils')
```

Comment:
I know that curl comes as transitive dependency of git, but maybe we should add explicitely?

Reply:
WFM